code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -711,10 +711,10 @@
 // Gives a lethal disease to the target.
 /mob/living/carbon/human/proc/vampire_diseasedtouch()
 	set category = "Vampire"
-	set name = "Diseased Touch (100)"
+	set name = "Diseased Touch (200)"
```

Comment:
Bad fix. This would be a rebalance of the power, which is unnecessary at present.

Change line 750 to `vampire.use_blood(100)` instead.

Reply:
updated